### ISSUE TITLE
model loss and total loss

### ISSUE THREAD

**Issue by caoyangcr7** [argman/EAST#195]
when I train my own data, at the beginning of training, the model loss is 0.01 ,the total loss is 0.04....and the model loss never changed! my own data is the same as ICDAR format like (x1,y1,x2,y2,x3,y3,x4,y4,label). As a result , after training, I get ckpt files but it didn't work,it can't detect the text in the images....So anyone meets this issue?

**Comment by Akshaypatil15** [comment]
Getting similar kind of issue for model loss and total loss, from beginning of the training I am getting this rates.
 `Step 049500, model loss 0.0100, total loss 0.0117, 4.64 seconds/step, 0.86 examples/second`

**Comment by IngleJaya95** [comment]
@caoyangcr7 I facing same issue on my training data, Can you suggest me how you resolve it, as you closed your issue ?

**Comment by caoyangcr7** [comment]
> @caoyangcr7 I facing same issue on my training data, Can you suggest me how you resolve it, as you closed your issue ?
you can refer the reply I just comment here. May that help you.

**Comment by JackeyDeng** [comment]
I met this issue too. And i solved the problem.it is because I give the wrong value to text_tags in icdar.py. 
text_tags.append(True) in function load_annoataion(p):True means the mask,and it will not be used to train,if you want to train all your polys, they should all be set as False.

**Comment by caoyangcr7** [comment]
@JackeyDeng  Yes, I found that you are right.

**Comment by caoyangcr7** [comment]
@JackeyDeng Thanks for your correction.

**Comment by decajcd** [comment]
> I met this issue too. And i solved the problem.it is because I give the wrong value to text_tags in icdar.py.
> text_tags.append(True) in function load_annoataion(p):True means the mask,and it will not be used to train,if you want to train all your polys, they should all be set as False.
After changed this paramater,what‘s the loss value?

**Comment by sdzbft** [comment]
> I met this issue too. And i solved the problem.it is because I give the wrong value to text_tags in icdar.py.
> text_tags.append(True) in function load_annoataion(p):True means the mask,and it will not be used to train,if you want to train all your polys, they should all be set as False.

thanks a lot for your correct reply
